.github/workflows/automate_changeset_feedback.yml

```diff
@@ -37,7 +37,10 @@ jobs:
           rm -f generate.js
           wget -O generate.js https://raw.githubusercontent.com/backstage/backstage/master/scripts/generate-changeset-feedback.js 1>&2
           node generate.js FETCH_HEAD > feedback.txt
-
+        # This command will get the last published version of @backstage/repo-tools and use that version
+        # (NPX doesn't work because it uses the current source)
+        # should be uncommented only after having one version with `changeset-feedback` published
+        # run: yarn dlx @backstage/repo-tools@latest changeset-feedback FETCH_HEAD > feedback.txt
```

Comment:
Hmm, not sure that this is safe tbh. I think it would be possible to reconfigure yarn to point to a private registry where you override `@backstage/repo-tools` to inject your own code. I think we may need to wrap this up in a custom action or something like that

Reply:
To expand a bit more on this, just shipped https://github.com/backstage/backstage/pull/15043. We can't even shell out to `yarn` since the implementation of that lives in the repo.